Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
Please create a branch for the 20.04 version so that we can keep it around on DockerHub.  See the other branches for an example: https://github.com/level12/python-test-multi/branches